python/basic-example/README.md

```diff
@@ -0,0 +1,3 @@
+# Basic Example
+
+The code in `test.py` is taken from [Step 2 of the Python SDK Getting Started Guide](https://docs.dagger.io/sdk/python/628797/get-started#step-2-create-a-dagger-client-in-python)
```

Comment:
Would be nice to have quick example of how to run here:

- create venv, activate
- `python test.py`